ampd/Dockerfile

```diff
@@ -22,7 +22,7 @@ COPY ./ampd/build.rs ./ampd/build.rs
 RUN cargo install --locked --path ./ampd
 
 FROM debian:bookworm-slim AS runner
-RUN apt update && apt install libssl3
+RUN apt-get update && apt-get install -y ca-certificates libssl3 && rm -rf /var/lib/apt/lists/*
```

Comment:
I think it's recommended to use apt-get.

> i don't understand why this didn't result in the handler throwing an error instead of a silent failure. can you address that?
we have this practice for batched requests, can revisit this later or add a error log at least

Reply:
if it's a small change, can we include the error reporting fix here? it'll save a lot of time in the future. looks like we just need to raise the error [here](https://github.com/axelarnetwork/axelar-amplifier/blob/d971f0efbfebb29c8715ac2169b5f2750dddb500/ampd/src/stellar/http_client.rs#L104)?